securedrop/sass/modules/_flash.sass

```diff
@@ -38,11 +38,25 @@
       i
         color: #D62727
 
+    &.important-header
+      margin: auto 6px
+      color: $color_urgent_coral
+
     &.important
-      border: 1px solid #EBDCEB
-      background: #FDFAFD
+      border-top: 1px solid #ece6e7
+      border-left: 6px solid $color_urgent_coral
+      border-bottom: 1px solid $color_urgent_coral
+      border-right: 1px solid #ece6e7
+      background-color: #FDFDFD
+      color: #383838
+      align-items: center
```

Comment:
Consider adding a bit of `margin-bottom` here to give some breathing room (see the spacing between message and logo in Nina's mockup); I'd go with ~25px for starters.

Reply:
added margin and moved styling changes from .flash to .flash.important in 0acdd39

Reply:
✔️ 0acdd39